content/posts/linux/Desktop Linux Hardening.md

```diff
@@ -244,6 +244,13 @@ You could also set your default firewall zone to drop packets. To implement this
 firewall-cmd --set-default-zone=drop
 firewall-cmd --add-protocol=ipv6-icmp --permanent
 firewall-cmd --add-service=dhcpv6-client --permanent
+firewall-cmd --reload
+```
+
+On some distributions, `--set-default-zone` might be whitelisted by default which leads to bypasses. To disable this, turn on lockdown mode for `firewalld`:
+
+```
+sudo firewall-cmd --lockdown-on
 ```
```

Comment:
```suggestion
On some distributions, it may be possible for unauthorized users or applications to make firewall changes through polkit. To disable this, enable firewalld _lockdown mode_ with `sudo firewall-cmd --lockdown-on`.
```